templates/desktop/index.html

```diff
@@ -1,98 +1,94 @@
+      <a href="http://cn.ubuntu.com/download" class="p-button">下载</a>
+      <a class="p-link--external" href="去Ubuntu开发者网站看看"></a>
     </div>
-</div>
-{% endif %}
+  </div>
+</section>
 
-{# ================ Who is behind Ubuntu? ================ #}
-{% if who_is_behind_kylin %}
-<div id="ubuntu-kylin-backers" class="row">
-    <div class="six-col">
-        {% include "templates/_cms-block.html" with block_content=who_is_behind_kylin %}
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <h3>Ubuntu Kylin</h3>
+      <p>优麒麟(英文名为Ubuntu Kylin)是基于Ubuntu的一款官方衍生版。 它是一款专门为中国市场打造的免费操作系统，而且它已经被录入中国政府采购条例名单中。它包括Ubuntu用户期待的各种功能，并配有必备的中文软件及程序。</p>
+      <a href="http://cn.ubuntu.com/download" class="p-button">下载Ubuntu</a>
+      <a href="http://www.ubuntukylin.com/" class="p-link--external"></a>
     </div>
-    <div class="six-col text-center last-col">
-            <ul class="no-bullets clearfix align-center">
-                <li><img src="{{ STATIC_URL }}img/desktop/logo-canonical-288.png" alt="Canonical" width="288" height="40"></li>
-                <li><img src="{{ STATIC_URL }}img/desktop/logo-csip-180.png" alt="CSIP" width="180" height="71"></li>
-                <li><img src="{{ STATIC_URL }}img/desktop/logo-nudt-288.png" alt="NUDT" width="288" height="72"></li>
-            </ul>
-        </div>
-</div>
-{% endif %}
+    <div class="col-6">
+      <img src="{{ STATIC_URL }}img/homepage/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+    <h3>优麒麟是由</h3>
+    <p><a class="p-link--external" href="http://www.canonical.com">Canonical</a>，<a class="p-link--external" href="http://www.csip.org.cn">工业和信息化部软件与集成电路促进中心（CSIP）</a>以及<a class="p-link--external" href="http://www.nudt.edu.cn">国防科学技术大学（NUDT）</a>携手打造。
+    目前，开发团队正在与开源社区合作打造Ubuntu的核心功能，以更好地适应中国用户。 除了Ubuntu软件中心推出的本土化版本以外，优麒麟还带最常用的中文程序。</p>
+  </div>
+    <div class="col-6">
+    <ul class="p-list u-align--center">
+      <li><img src="{{ STATIC_URL }}img/desktop/logo-canonical-288.png" alt="Canonical" width="288" height="40"></li>
+      <li><img src="{{ STATIC_URL }}img/desktop/logo-csip-180.png" alt="CSIP" width="180" height="71"></li>
+      <li><img src="{{ STATIC_URL }}img/desktop/logo-nudt-288.png" alt="NUDT" width="288" height="72"></li>
+    </ul>
+  </div>
+  </div>
+</section>
 
-{# ================ Whats new? ================ #}
-{% if features %}
 <div id="row-whats-new" class="row row-grey">
     <div class="seven-col">
-        {% include "templates/_cms-block.html" with block_content=features %}
+
     </div>
 </div>
-{% endif %}
 
-{# ================ Find out more ================ #}
-{% if find_out_more %}
 <div id="row-more" class="row no-border">
-    <div class="eight-col">
-        {% include "templates/_cms-block.html" with block_content=find_out_more %}
+    <div class="col-8">
+
```

Comment:
Empty div?

Reply:
I think this is a previous iteration of the code?